### PR TITLE
Submission 1 changes

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
Key checks: 
- fix modal toast display:
      -  https://deploy-preview-12--design-with-ira.netlify.app/pages/components/modal.html 
      - https://deploy-preview-12--design-with-ira.netlify.app/pages/components/snackbar.html
- Installation Docs: https://deploy-preview-12--design-with-ira.netlify.app/pages/installation.html
- stacked List: https://deploy-preview-12--design-with-ira.netlify.app/pages/components/list.html

### CODE REVIEW

File: pages/components/button.html [hunk 1]
```diff
@@ -23,90 +23,96 @@
...
 
-      <li class="nav-list-item"> <a href=/pages/about.html>About </a> </li> <li class="nav-list-item"> <a
-            href=/pages/docs.html>Documentation </a> </li> <li class="nav-list-item"> <a
-              href=/pages/installation.html>Installation </a> </li> <li class="nav-list-item"> <a
-                href="https://github.com/HarshnaNHaswani/Ira">GitHub</a></li>
-    </ul>
-  </nav>
+      <li class="nav-list-item"> <a href=/pages/about.html>About </a> </li> <li class="nav-list-item">
+          <button class="hamburger">Documentation</button>
+          <aside>
```
Comment (ivinayakg) [inline pages/components/button.html:28]: nice use of semantic html

**Review by ivinayakg — APPROVED** [review]
Yep, its nice and ready to merge

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Comment by HarshnaNHaswani** [comment]
Thanks for your time :)

**Comment by HarshnaNHaswani** [comment]
@yashpatel1797 @ivinayakg Thanks for your time and review!

**Pull request merged**